packages/webapp/src/webview/state/reducers.ts

```diff
@@ -231,6 +234,26 @@ function restartAnswerReducer(newState: AppState): AppState {
     return newState;
 }
 
+/**
+ * Go to home page.
+ *
+ * @param newState - already cloned state that is modified and returned
+ * @returns new state with changes
+ */
+function goToHomePageReducer(newState: AppState): AppState {
+    newState.isHome = true;
```

Comment:
See my other comment, this would become
```suggestion
    newState. activeScreen = 'Home';
```